pkg/endpoint/endpoint.go

```diff
@@ -1025,8 +1022,8 @@ func (e *Endpoint) GetPolicyModel() *models.EndpointPolicyStatus {
 	sortProxyStats(proxyStats)
 
 	mdl := &models.EndpointPolicy{
-		ID:                       int64(e.Consumable.ID),
-		Build:                    int64(e.Consumable.Iteration),
+		ID:                       int64(e.SecurityIdentity.ID),
+		Build:                    int64(e.Iteration),
```

Comment:
This is definitely wrong. You need to store a policy revision in the realizedState and inherit the revision from the desiredState when it was fully implemented.

Reply:
This is no worse than what we have today, no? Currently, we arbitrarily produce the model on GetModel, but we increment Iteration at the start of builds (I think, in any case, I don't think the PR changed when Iteration was incremented).

Reply:
Yes but previously the reported realized and desired state always matched. Now they no longer match but the revision does not indicate this.

Reply:
I use `nextPolicyRevision` now.